R/read_txt_Renishaw.R

```diff
@@ -164,48 +164,47 @@ read_txt_Renishaw <- function(file = stop("file is required"),
 
 hySpc.testthat::test(read_txt_Renishaw) <- function() {
   context("read_txt_Renishaw")
+  path <- system.file("extdata/fileio/txt.Renishaw", package="hySpc.read.txt")
+  paracetamol <- paste0(path, "/paracetamol.txt")
+  laser <- paste0(path, "/laser.txt.gz")
+  chondro <- paste0(path, "/chondro.txt")
 
   test_that("single spectrum", {
-    skip("TODO: adapt to new package")
-    tmp <- read_txt_Renishaw("fileio/txt.Renishaw/paracetamol.txt", "spc")
+    tmp <- read_txt_Renishaw(paracetamol, "spc")
     expect_equal(dim(tmp), c(nrow = 1L, ncol = 2L, nwl = 4064L))
   })
 
   test_that("time series spectrum, gzipped", {
-    skip("TODO: adapt to new package")
-    tmp <- read_txt_Renishaw("fileio/txt.Renishaw/laser.txt.gz", "ts")
+    tmp <- read_txt_Renishaw(laser, "ts")
     expect_equal(dim(tmp), c(nrow = 84L, ncol = 3L, nwl = 140L))
     expect_equal(colnames(tmp), c("t", "spc", "filename"))
   })
 
   test_that("map (= default)", {
-    skip("TODO: adapt to new package")
-    tmp <- read_txt_Renishaw("fileio/txt.Renishaw/chondro.txt", "xyspc")
+    tmp <- read_txt_Renishaw(chondro, "xyspc")
```

Comment:
Do not use `chondro` as it was removed from `hyperSpec`. Use `faux_cell` instead (in this and all other places where `chondro` is used)

Reply:
There is no such testing file called faux_cell in the testing data folder.

Reply:
Sorry, I was confused by the name `chondro` as many years it meant a different thing.

Please add `f_` to variable names that contain file paths, e.g. `f_chondro`